<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddReaderDialog</class>
 <widget class="QDialog" name="AddReaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录入读者</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>录入读者</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>读者编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ridEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rnameEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sex_man">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>男</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sex_woman">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>普通用户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高级用户</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="submitBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
